stack/tasks/create-a-m-o.js

```diff
@@ -0,0 +1,234 @@
+const fs = require('fs');
+const mkdirp = require('mkdirp'); // eslint-disable-line import/no-extraneous-dependencies
+
+const CWD = process.cwd();
+
+process.stdin.setEncoding('utf8');
+
+// TODO, evaluate https://github.com/chalk/chalk
+
+// ref: https://stackoverflow.com/questions/9781218/how-to-change-node-jss-console-font-color
+console.log('\x1b[40m', '\x1b[36m', // eslint-disable-line
+  `
+  Hello Dear developer, thank you for contributing with us. 😊
+
+  I will help you create a new web component 😎.
+
+  You can choose between ATOM📗, MOLECULE📘 and ORGANISM📙.
+
+  As a general guideline, an ATOM📗 is indivisible and is the smallest component.
+  I won't make sense to use it on its own, but is a essential builing block. An ATOM📗 should not have dependencies to other elements
+  An ORGANISM📙 is the finished and ready to use component. It must have at least one element as dependency.
+  A MOLECULE📘 is a not completly finished component and can be resused somewhere else. It must contain at least one ATOM📗.
+
+  Now, please tell me what do you wan to create
+  `,
+);
+
+let element = '';
+
+const mapElement = {
+  a: 'ATOM📗',
+  m: 'MOLECULE📘',
+  o: 'ORGANISM📙',
+};
+
+
+const capitalizeFirstLetter = string => string.charAt(0).toUpperCase() + string.slice(1);
+
+const displayNameText = () => {
+  console.log('\x1b[40m', '\x1b[37m', // eslint-disable-line
+    `
+  Please enter the name of the new ${mapElement[element]} ( something that make sense 😉 ).
+    `,
+  );
+};
+
+const displayElementSelector = () => {
+  console.log('\x1b[40m', '\x1b[37m', // eslint-disable-line
+    `
+  Press:
+
+  1 for ATOM 📗
+  2 for MOLECULE 📘
+  3 for ORGANISM 📙
+    `,
+  );
+};
+
+const writeIndexJs = (path, _name) => {
+  fs.writeFileSync(
+    `${path}/index.js`,
+    `import { BaseComponentGlobal } from '../_abstract/component-types';
+// import the styles used for this component
```

Comment:
maybe we can find a nice solution with template tag function to handle leading whitespace

Reply:
ohooo
https://www.npmjs.com/package/outdent